mod_reforged/hooks/skills/actives/throw_dirt_skill.nut

```diff
@@ -0,0 +1,22 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/throw_dirt_skill", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		this.m.Description = "This character hides a substantial amount of sand in their pockets, ready to be unleashed on an attacker.";
```

Comment:
`unleashed` isn't appropriate here for throwing sand. Similarly `attacker`. Could say `ready to be thrown at someone.`